### ISSUE TITLE
The end result is ugly ...

### ISSUE THREAD

**Issue by ThomaLa** [ThomaLa/group-4-9-attendance_tracking_system#7]
... but does the job. For the moment. But when it gets complicated, we might want to change its layout for ease of use, no?

**Comment by aynesh** [comment]
I guess for first few weeks as discussed yesterday we should focus on functionality ....

**Comment by ThomaLa** [comment]
Yeah of course, I also thought this to be a rather long-term issue with low priority :)
